packages/fe-mockserver-core/test/unit/v4/services/formSample/FormTemplate.cds

```diff
@@ -12,7 +12,8 @@ entity FormRoot {
         LastName     : String  @Common.Label :                                'Last Name';
         DateOfBirth  : Date    @Common.Label :                                'Date of Birth';
         EmailAddress : String  @Communication.IsEmailAddress  @Common.Label : 'Email Address';
-        _Elements    : Composition of many SubElements;
+        _Elements    :Association to many SubElements
```

Comment:
I wonder why SubElements is still a draft node if it is declared as an association. Maybe a special case in the CDS compiler? Anyway, it has the DraftNode annotation, so it is ok from a test perspective.

Reply:
Interesting i didn't even think of that, having it as composition remove the partner navigation so i'm lost